react/features/conference/components/web/Subject.js

```diff
@@ -2,9 +2,12 @@
 
 import React, { Component } from 'react';
 
+import ParticipantsCount from './ParticipantsCount';
```

Comment:
The group of imports from the same feature should be placed after the group with imports from other features.